src/motivating_scenario.md

```diff
@@ -70,7 +72,7 @@ A `Resource` in this scenario is effectively a filename, from the following dire
   employee_file0.avro
   employee_file1.avro
 ```
-These are provided by a _data catalogue_ which is distinct from the target _data source_, and can be tagged with additional metadata.
+This metadata comes from a _data catalogue_, but it could equally work by doing a directory listing of the underlying data store and/or using shadow files for metadata info about each resource. 
```

Comment:
```suggestion
Ideally this metadata comes from a _data catalogue_, but it can equally work by doing a directory listing of the underlying data store and/or using shadow files for metadata info about each resource. 
```